### PR TITLE
refactor: show assignee name on select input in edit mode

### PR DESCRIPTION
**Pull request by bharati-21**
### Issue:
https://github.com/Real-Dev-Squad/website-status/issues/758

### Description:
- For assigned tasks in edit mode, the name of the assignee was not visible and had to be resolved using the assignee's image
- With this change, the assignee name appears on the user select drop down in the edit mode

### Anthing you would like to inform the reviewer about:
N/A

### Dev Tested:
- [x] Yes

### Images/video of the change:
image

### Follow-up Issues (if any)
N/A

### Test coverage stats
image

### CODE REVIEW

**Review by sahsisunny — APPROVED** [review]

File: __tests__/Unit/Components/Tasks/Card.test.tsx
Comment (Pratiyushkumar) [inline __tests__/Unit/Components/Tasks/Card.test.tsx:1]: is it possible to test that if cardDetails.assignee value is empty then what would happen

Comment (bharati-21) [reply __tests__/Unit/Components/Tasks/Card.test.tsx:1]: Done

**Review by vinit717 — APPROVED** [review]
LGTM

**Review by Pratiyushkumar — APPROVED** [review]
LGTM

**Pull request merged**